netlify/netlify-plugin-nextjs
# fix: correct root redirect when trailingSlash = false

### Summary

The root level redirect when NEXT_LOCALE is set was incorrectly defined when trailingSlash was set to false.



### Test plan

1. Set a test site to trailingSlash = false
2. Check that there is no redirect created with `from = ''` 

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Fixes #872 

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
